### PR TITLE
Update DL

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - add more detail & contexts to dl_

---

### What kind of change does this PR introduce?

- DL credential update.

<br/>

### What is the current behavior?

- More information needed for DL example.

<br/>

### What is the new behavior?

- More credential subject values, types, and context are added from the [W3C verifiable DL vocabulary example](https://w3c-ccg.github.io/vdl-vocab/#example-a-iso18013-5-mdl-expressed-as-a-w3c-verifiable-credential)

<br/>

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

<br/>

### How has this been tested?

- Credential has been issued and viewed locally.

<br/>

### Screenshots: n/a

### CODE REVIEW

File: credentials/utopia-dl/credential.json [hunk 1]
```diff
@@ -1,21 +1,56 @@
...
+  "description": "A license granting driving privileges in Utopia.",
+  "credentialSubject": {
+    "id": "did:example:12347abcd",
+    "type": "LicensedDriver",
+    "driversLicense": {
+      "type": "Iso18013DriversLicense",
+      "document_number": "542426814",
+      "family_name": "SMITH",
+      "given_name": "JOHN",
+      "portrait": "https://vcplayground.org/examples/utopia-dl/dl-male-portrait.png",
```
Comment (BigBlueHat) [inline credentials/utopia-dl/credential.json:27]:
```suggestion
      "image": "https://vcplayground.org/examples/utopia-dl/dl-male-portrait.png",
```

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
@jameseaster can you add a `resident_address` field to this credential? Annoyingly it's just an undifferentiated string in an mDL...

Here's the address to use:
123 Main Street
Anyville, UTO F87P2

**Comment by jameseaster** [comment]
@BigBlueHat 
Overall, huge improvement on load time! Looks like a few text issues with the dates,  looking killer as always though!

For reference:

> SVG with values
>
> svg

**Comment by BigBlueHat** [comment]
@jameseaster give this another test if you would. I've inlined the https://fonts.google.com/specimen/Manrope font into the SVG (via CSS).

**Comment by BigBlueHat** [comment]
Here's what things should look like when rendered:
image

Remaining issues include:
[ ] date formatting (see `exp`)
[ ] missing values (notably `hgt` will need additional parsing/formatting/conversion)
[ ] mapping the `sex` integer from the upstream spec to a meaningful value

**Comment by BigBlueHat** [comment]
Latest and geatest.
image

Font issues all fixed. Date format "cheated" (for now) by only storying the date (sans time) in the credential info. Other missing data was added. The `sex` value is still not mapped to anything meaningful...as we need to discuss how we want to do that.

**Review by BigBlueHat — APPROVED** [review]
@jameseaster this probably needs your review...even though GitHub doesn't realize PRs might have multiple authors who need to review each others code...yet(?).

**Comment by jameseaster** [comment]
@BigBlueHat The DL is unable to be issued in the playground. I'm receiving this error:
```
Unable to generate/issue credential: Unable to issue credential on selected backend: "expirationDate" must be a valid date: 2030-04-19
```

**Comment by davidlehn** [comment]
4MB json, 2.8MB svg, 168KB image.  Any chance for optimization here?

**Comment by jameseaster** [comment]
> 4MB json, 2.8MB svg, 168KB image. Any chance for optimization here?

@davidlehn @BigBlueHat @dlongley 
Could be helpful to link to an image property with a url, rather than using a massive base64 string. I know there are some security questions here, but seems like a quick win to help minimize the json for the credential.

**Comment by BigBlueHat** [comment]
> This seems like a new error, and I can see the value in the credential `"expirationDate": "2030-04-19T12:00:00-06:00"`

It's because I reduced the `expirationDate` value down to `2030-04-19` so it looks good in the design... I don't understand why that's not being treated as a valid date...other than we've gone a bit farther than ISO 8601 dates in our requirements. @dlongley thoughts?

**Comment by dlongley** [comment]
@BigBlueHat, yeah, IIRC, it needs to be a datetime in the spec, not just a date. At least, I think that's a requirement for `validUntil` in VC 2.0. Ultimately, we want the render method to be able to handle this -- so we need to find a solution for that.

**Comment by BigBlueHat** [comment]
> @BigBlueHat, yeah, IIRC, it needs to be a datetime in the spec, not just a date. At least, I think that's a requirement for `validUntil` in VC 2.0. Ultimately, we want the render method to be able to handle this -- so we need to find a solution for that.

Yeah, `validUntil` is more than an ISO 8601, apparently. Specifically, it's https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp

**Comment by BigBlueHat** [comment]
@jameseaster could you give this another try on your side?

**Comment by jameseaster** [comment]
@BigBlueHat It is unable to render the displays b/c of the change to raw svg

**Comment by davidlehn** [comment]
You can lint the JSON with the json-ld playground or jsonld-cli.  It thinks the inline svg is a relative URL.  It looks like it needs to be URL encoded (which will increase the size).  Consider first running source through svgo to optimize more bytes out.

**Comment by jameseaster** [comment]
Rendered locally with the compressed svg stored in `template` - LGTM

Thanks @BigBlueHat!

**Pull request merged**